package/MDAnalysis/core/AtomGroup.py

```diff
@@ -3087,6 +3087,8 @@ def select_atoms(self, selstr, *othersel, **selgroups):
         .. versionchanged:: 0.13.1
            Added implicit OR syntax to field and range selections
         """
+        if selstr is "":
+            return None
```

Comment:
This was a result of the change that I proposed in the (admittedly rather long) comment I posted yesterday about the new MemoryReader behaviour. In item 5. of that comment, I suggested that it would be convenient if we could somehow deal with the concept of making _no_ selection. I recognise that this is a rather fundamental change - and you might not agree to it - but please see the entire post above.